tests/system_tests/plans.json

```diff
@@ -89,7 +82,7 @@
                             },
                             "radius": {
                                 "description": "Radius of the circle",
-                                "exclusiveMinimum": 0,
+                                "exclusiveMinimum": 0.0,
```

Comment:
I do not know why these changes have appeared here just from adjusting handling of `| None`, as they do not allow None or defaults... I have always been confused why they got set to integer 0 instead of float 0.0, and I think this is more correct but why!?

```python
StrictConfig: ConfigDict = {"extra": "forbid", "arbitrary_types_allowed": True}

@dataclass(config=StrictConfig)
class Circle(Region[Axis]):
      [...]
      radius: float = Field(description="Radius of the circle", gt=0.0)

```